waspc/src/Generator/Entity/EntityForm.hs

```diff
@@ -29,6 +29,7 @@ data EntityFormTemplateData = EntityFormTemplateData
     , _formFields :: ![FormFieldTemplateData]
     -- Submit
     , _showSubmitButton :: !Bool
+    , _isXRayModeEnabled :: !Bool
```

Comment:
Aha, this is because this one does not take whole wasp object right?
Slightly sucks that we have to know about it here, but I am not sure what would be the other way, ideally it would somehow be completely separated thing but I guess that would require too much work for now and changing architecture and similar.

Reply:
yes, it would be nicer to maybe have some additional top level data layer which contains this "meta" information.